atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -322,7 +330,7 @@ OptionalLong getPreviousMinimumSweptTimestamp() {
                     Optional<TableReference> nextTable = nextTableToSweepProvider.chooseNextTableToSweep(
                             tx, sweepRunner.getConservativeSweepTimestamp());
                     if (nextTable.isPresent()) {
-                        log.debug("Now starting to sweep {}.", nextTable);
+                        log.info("Now starting to sweep {}.", UnsafeArg.of("next table", nextTable));
```

Comment:
Leave at debug especially as table is not collected.. finished sweeping info is enough to confirm sweep has been running